src/BenchmarkDotNet.Core/Environments/HostEnvironmentInfo.cs

```diff
@@ -71,7 +75,9 @@ protected HostEnvironmentInfo()
             BenchmarkDotNetVersion = GetBenchmarkDotNetVersion();
             OsVersion = new Lazy<string>(RuntimeInformation.GetOsVersion);
             ProcessorName = new Lazy<string>(RuntimeInformation.GetProcessorName);
-            ProcessorCount = Environment.ProcessorCount;
+            LogicalCoresCount = Environment.ProcessorCount;
+            PhysicalCoresCount = new Lazy<int?>(RuntimeInformation.GetPhysicalCoresCount);
+            PhysicalProcessorsCount = new Lazy<int?>(RuntimeInformation.GetPhysicalProcessorsCount);
```

Comment:
I think it would be better to use names like `LogicalCoreCount`, `PhysicalCoreCount`, `PhysicalProcessorCount` (without `s`).